.gitignore

```diff
@@ -12,5 +12,5 @@
 !/log/.keep
 /tmp
 .byebug_history
-
+/public/headshots
```

Comment:
Good idea, otherwise there would be like 50 pictures of you in this pull request. :sparkles: